clients/go-mobile/client.go

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2022 ANSSI
```

Comment:
I can't find where we check that the AK is a genuine TPM key rather than a fake one from a virtual TPM.